### PR TITLE
Updating of packages, removing of unneeded packages, misc. cleanup

### PR DESCRIPTION
**Pull request by cjihrig**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,9 +1,13 @@
 {
  "extends": "./node_modules/lab/lib/linters/eslint/.eslintrc",
- "parser": "babel-eslint",
+ "env": {
```
Comment (arb) [inline .eslintrc]: What uses this?

Comment (cjihrig) [inline .eslintrc]: The `es6` part is to get the ES6 features turned on. The `node` part is just for good measure.

Comment (arb) [inline .eslintrc]: Oh... I thought I was looking at "package.json"... Silly me.

**Comment by cjihrig** [comment]
@arb ping

**Pull request merged**